dev-docs/source/SystemTests.rst

```diff
@@ -360,6 +360,31 @@ by using the ``-c`` or ``--clean`` option, e.g.
 This is useful if some old data is left over from a previous run,
 where some tests were not cleanly exited.
 
+Processing test failures
+------------------------
+If any of the tests fail, a json file (FAILED-systemtests.json) containing information on the failed tests
+will be outputted. If the information was available, each entry in the file will contain the name of the failed test,
+the mismatch file generated and the name and location of the file it was compared against (the reference).
+The information can be used to automatically generate replacement references using the addTestData.py python script
```

Comment:
We could add a link to the `addTestData.py` script as is done in the release checklist to some of the helper scripts